Documentation/feature-band-source-building.md

```diff
@@ -0,0 +1,825 @@
+- 3xx builds always use 2xx tooling (never the 3xx SDK).
+- 3xx artifacts are only used for distribution, never as build inputs for
+  other bands.
+- Servicing builds use source-built SDKs from the previous releases of the
+  same band (or 2xx for 3xx).
+- For bootstrapping of 2xx/3xx feature bands, stage 2 does not require the 1xx
+  shared components as input. The artifacts produced by stage 1 contain all
+  required packages.
+
+## Build Command Arguments
+
+Feature band builds make use of these relevant command-line arguments:
+
+- `--with-packages` - Provides PSB artifacts
+- `--with-shared-components` - Provides shared component artifacts from 1xx build
+- `--with-sdk` - Specifies the SDK to use for building
+
+## Distro Maintainer Workflows
+
+The following sections describe the workflows for different scenarios.
+
+### Input Artifacts Summary
+
+**For 1xx band builds:**
+
+- **Bootstrap**:
+  - PSB artifacts: Microsoft-built previous N.0.1xx release
+  - SDK: Microsoft-built previous N.0.1xx release
+- **Servicing**:
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+
+**For 2xx band builds:**
+
+- **Initial release (N.0.200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.1xx release
+  - SDK: Source-built previous N.0.1xx release
+- **Bootstrap initial release (N.0.200)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.1xx release
+  - SDK: Microsoft-built previous N.0.1xx release
+- **Bootstrap (N.0.201+)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.2xx release
+  - SDK: Microsoft-built previous N.0.2xx release
+- **Servicing (N.0.201+)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.2xx release
+  - SDK: Source-built previous N.0.2xx release
+
+**For 3xx band builds:**
+
+- **Initial release (N.0.300)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.2xx release
+  - SDK: Source-built previous N.0.2xx release
+- **Bootstrap (N.0.300)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.2xx release
+  - SDK: Microsoft-built previous N.0.2xx release
+- **Bootstrap (N.0.301+)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Microsoft-built previous N.0.2xx release
+  - SDK: Microsoft-built previous N.0.2xx release
+- **Servicing (N.0.301+)**:
+  - Shared component artifacts: Source-built current N.0.1xx release
+  - PSB artifacts: Source-built previous N.0.2xx release
+  - SDK: Source-built previous N.0.2xx release
+
+### Scenario 1: 1xx Band Bootstrap
```

Comment:
I see this doc as being a useful reference at times.  Because of that I think have a table of contents would be useful mainly for these scenarios.

I also see the 'Input Artifacts Summary' as being redundant with these scenarios.  IMO the summary should be removed.  When I read it I wanted to see this information along side it.

Also IMO `scenario x` in the headings doesn't add value.  The descriptive name is what is useful.